MANIFEST.in

```diff
@@ -31,3 +31,7 @@ recursive-include etc *.yaml
 recursive-include helm *.md
 recursive-include helm *.tpl
 include *.yaml
+
+# benchmarks
```

Comment:
Instead of using a new `benchmarks` directory, we could put the script to `scripts` where the helper scripts usually live.  And we could call it `reana-bench` for shorter.

But, perhaps more interesting would be to put it to `reana/reana-bench` similarly to `reana-dev`, so that it can grow... We would have `reana-dev` for developers, `reana-dev` for benchmarking, and `reana-client` for regular users...

Reply:
I will move it to `scripts/` and rename it to `reana_bench.py` for now. We can work on packaging it as `reana/reana-bench` in the next steps.